bokehjs/src/lib/models/glyphs/webgl/regl_wrap.ts

```diff
@@ -151,43 +144,50 @@ function regl_solid_line(regl: any): ReglRenderFunction {
         buffer: regl.buffer(line_instance_geometry),
         divisor: 0,
       },
-      a_point_prev: {
-        buffer: regl.prop("points"),
-        divisor: 1,
-        offset: Float32Array.BYTES_PER_ELEMENT * 0,
-      },
-      a_point_start: {
-        buffer: regl.prop("points"),
-        divisor: 1,
-        offset: Float32Array.BYTES_PER_ELEMENT * 2,
-      },
-      a_point_end: {
-        buffer: regl.prop("points"),
-        divisor: 1,
-        offset: Float32Array.BYTES_PER_ELEMENT * 4,
-      },
-      a_point_next: {
-        buffer: regl.prop("points"),
-        divisor: 1,
-        offset: Float32Array.BYTES_PER_ELEMENT * 6,
+      a_point_prev: (_: Context, props: Props) => {
+        return {
+          buffer: regl.buffer(props.points),
+          divisor: 1,
+        }
+      },
+      a_point_start: (_: Context, props: Props) => {
+        return {
+          buffer: regl.buffer(props.points),
+          divisor: 1,
+          offset: Float32Array.BYTES_PER_ELEMENT * 2,
+        }
+      },
+      a_point_end: (_: Context, props: Props) => {
+        return {
+          buffer: regl.buffer(props.points),
+          divisor: 1,
+          offset: Float32Array.BYTES_PER_ELEMENT * 4,
+        }
+      },
+      a_point_next: (_: Context, props: Props) => {
+        return {
+          buffer: regl.buffer(props.points),
+          divisor: 1,
+          offset: Float32Array.BYTES_PER_ELEMENT * 6,
+        }
       },
     },
 
     uniforms: {
-      u_canvas_size: regl.prop('canvas_size'),
-      u_pixel_ratio: regl.prop('pixel_ratio'),
-      u_color: regl.prop('color'),
-      u_linewidth: regl.prop('linewidth'),
-      u_antialias: regl.prop('antialias'),
-      u_miter_limit: regl.prop('miter_limit'),
-      u_join_type: regl.prop('join_type'),
-      u_cap_type: regl.prop('cap_type'),
+      u_canvas_size: regl.prop<Props, 'canvas_size'>('canvas_size'),
```

Comment:
This overspecification is distracting, but given how regl's API is design, it's the best that can be done, at least without adding an abstraction layer on top of the base regl API.